presto-spi/src/main/java/io/prestosql/spi/connector/ConnectorMetadata.java

```diff
@@ -231,8 +231,9 @@ default TableStatistics getTableStatistics(ConnectorSession session, ConnectorTa
 
     /**
      * Creates a schema.
+     * @param principal TODO
      */
-    default void createSchema(ConnectorSession session, String schemaName, Map<String, Object> properties)
+    default void createSchema(ConnectorSession session, String schemaName, Map<String, Object> properties, PrestoPrincipal principal)
```

Comment:
s/principal/authorization or owner (but if you choose one thing, then please use coherently across the commit).

Reply:
At the metadata level, I guess "owner" is better. I will make this so.